addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -91,6 +91,15 @@
             </we-select>
             <we-input string="Input Placeholder" data-select-attribute="" data-attribute-name="placeholder"
                       data-apply-to='input[type="text"], input[type="email"], input[type="number"], textarea'/>
+            <we-input string="Default Value" data-select-textarea-default-value="" data-apply-to='textarea'/>
```

Comment:
- Can you avoid using single quoted string as xml attributes? It gives me nightmares
- Why not using select-property here?

Reply:
- Didn't meant to spook you ^^
- I wanted to but I have to set both value and textContent. It would work if I could set 2 values like with a JSON array but don't know if usefull to do that only for this.

Reply:
Why would you need to set `value`? Just `textContent` did not work?